main/integration-tests/__test__/api-tests/common/advanced-tests/rstudio/launch-rstudio-workspace.test.js

```diff
@@ -56,6 +56,17 @@ describe('Launch and terminate RStudio instance', () => {
           return workspace.id;
         }),
       );
+      const workspacesInAvailableState = response.filter(workspace => {
+        return workspace.envTypeId === rstudioEnvTypeId && workspace.status === 'COMPLETED';
+      });
+      console.log('COMPLETED (Available) workspaces', workspacesInAvailableState);
+      for (let i = 0; i < workspacesInAvailableState.length; i += 1) {
+        console.log(`Terminating ${workspacesInAvailableState[i].id}`);
+        // eslint-disable-next-line no-await-in-loop
+        await adminSession.resources.workspaceServiceCatalogs
+          .workspaceServiceCatalog(workspacesInAvailableState[i].id)
+          .delete();
+      }
```

Comment:
Feels a little odd that we are terminating workspaces in method `checkAllRstudioWorkspaceIsTerminated`. Can we move this to another method like `terminateAllRStudioWorkspacesInCompletedState`?

Reply:
Done. I've created a new method for terminating COMPLETED workspaces.